migrations/20250318120849_departition_af_collab.sql

```diff
@@ -0,0 +1,103 @@
+set statement_timeout to 3600000;
+
+-- create new uniform collab table
+create table af_collab_temp
+(
+    oid uuid not null primary key,
+    workspace_id  uuid not null references public.af_workspace on delete cascade,
+    owner_uid bigint not null,
+    partition_key integer not null,
+    len integer,
+    blob bytea not null,
+    deleted_at timestamp with time zone,
+    created_at timestamp with time zone default CURRENT_TIMESTAMP,
+    updated_at timestamp with time zone default CURRENT_TIMESTAMP not null,
+    indexed_at timestamp with time zone
+);
+
+-- create a table for collabs that have non-UUID object ids
+create table af_collab_conflicts
+(
+    oid text not null primary key,
+    workspace_id  uuid not null references public.af_workspace on delete cascade,
+    owner_uid bigint not null,
+    partition_key integer not null,
+    len integer,
+    blob bytea not null,
+    deleted_at timestamp with time zone,
+    created_at timestamp with time zone default CURRENT_TIMESTAMP,
+    updated_at timestamp with time zone default CURRENT_TIMESTAMP not null,
+    indexed_at timestamp with time zone
+);
+
+-- move non-UUID object ids to the new table (execution time: 31 secs)
+insert into af_collab_conflicts(oid, workspace_id, owner_uid, partition_key, len, blob, deleted_at, created_at, updated_at, indexed_at)
+select oid, workspace_id, owner_uid, partition_key, len, blob, deleted_at, created_at, updated_at, indexed_at
+from af_collab
+where oid !~ E'^[[:xdigit:]]{8}-([[:xdigit:]]{4}-){3}[[:xdigit:]]{12}$';
```

Comment:
Since we don't want to remove user data, we move all incompatible records to separate table. Operators will have to resolve these issues.